hyp3_gamma/water_mask.py

```diff
@@ -44,6 +46,7 @@ def create_water_mask(input_tif: str, output_tif: str):
     mask = geopandas.read_file(mask_location, mask=extent)
     with NamedTemporaryFile() as temp_file:
         mask.to_file(temp_file.name, driver='GeoJSON')
```

Comment:
The number of bytes can be returned with `os.stat(temp_file.name).st_size`, so guess if we update OGR_GEOJSON_MAX_OBJ_SIZE=(os.stat(temp_file.name.st_size)+(buffer number))/1000000, that could work? It's a bit clunky, though